server/go/pkg/server/database.go

```diff
@@ -0,0 +1,25 @@
+package server
+
+import (
+	"database/sql"
+
+	"github.com/go-sql-driver/mysql"
+)
+
+func newMysql(connStr string) (*sql.DB, error) {
+	dsn, err := mysql.ParseDSN(connStr)
+	if err != nil {
+		return nil, err
+	}
+
+	dsn.ParseTime = true
+
+	db, err := sql.Open("mysql", dsn.FormatDSN())
+	if err != nil {
+		return nil, err
+	}
+
+	db.SetMaxOpenConns(90)
```

Comment:
should we add a note in the README about this config?

Reply:
I'm going to remove this line altogether as the default (unlimited) is a better choice.